wire-ios-sync-engine/Tests/TestsPlans/AllTests.xctestplan

```diff
@@ -9,6 +9,7 @@
     }
   ],
   "defaultOptions" : {
+    "codeCoverage" : false,
```

Comment:
Did you confirm that this was necessary?

Reply:
not completely sure, but I think it does not hurt to keep it disabled for now since we don't collect code coverage for now